framework/src/Volo.Abp.Caching/Volo/Abp/Caching/DistributedCache.cs

```diff
@@ -172,33 +193,57 @@ protected virtual void SetDefaultOptions()
             {
                 return GetManyFallback(
                     keyArray,
+                    considerUow,
                     hideErrors
                 );
             }
 
+            var cachedValues = new List<KeyValuePair<TCacheKey, TCacheItem>>();
```

Comment:
Move into `if (ShouldConsiderUow(considerUow))` block

Reply:
The `cachedValues `  variable will be used later.